scaleph-common/src/main/java/cn/sliew/scaleph/common/dict/flink/FlinkCatalogType.java

```diff
@@ -0,0 +1,40 @@
+package cn.sliew.scaleph.common.dict.flink;
+
+import cn.sliew.scaleph.common.dict.DictInstance;
+import com.baomidou.mybatisplus.annotation.EnumValue;
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import java.util.Arrays;
+
```

Comment:
lake @JsonFormat(shape = JsonFormat.Shape.OBJECT), which take effect when jackson serialize DictInstance class and then return {"label": "foo", "value": "bar"}